config/schedule.rb

```diff
@@ -5,3 +5,7 @@
 every :day, at: '9:00am' do
   runner 'Assignment.send_reminders!'
 end
+
+every :day, at: '4:00am' do
+  runner 'Roster.find_each { |roster| RosterMailer.with(roster: roster).open_dates_alert.deliver_now }'
```

Comment:
I don't really love having anything more than a task/method call encoded in these runner strings. It brings application logic out of the application and also makes it untestable.

For these types of things I personally like writing `Job`s (say, a `CheckUncoveredDatesJob`) and calling them here like

```suggestion
  runner 'CheckUncoveredDatesJob.perform_now'
```

both making it testable and encapsulating it into a nice unit of work.